packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,770 @@
+        "lastUpdated": "2019-10-09T19:38:18.000Z",
+        "complexity": {
+            "minLength": 8,
+            "minLowerCase": 1,
+            "minUpperCase": 1,
+            "minNumbers": 1,
+            "minSymbols": 0
+        },
+        "exclude": {
+            "common": {
+                "source": {
+                    "type": "okta"
+                }
+            },
+            "attributeCriteria": []
+        },
+        "_links": {
+            "self": {
+                "href": "https://{yourOktaDomain}/api/v1/org/factors/okta_password/profiles/fpr3qbfxjK6iECycn0g4/features/fpf3qbixISW8mRVb00g4",
+                "hints": {
+                    "allow": [
+                        "GET",
+                        "PUT",
+                        "DELETE"
+                    ]
+                }
+            }
+        }
+    }
+```
+
+#### String Validation Factor Profile Feature Properties
+
+| Property                     | Type                                                                                                               | Description                                                  |
+| ---------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------ |
+| `_links`                     | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06)                                                     | Link relations for this object                               |
+| `complexity.minLength`       | Integer                                                                                                            | Minimum length required.                                     |
+| `complexity.minLowerCase`    | Integer                                                                                                            | Minimum number of lower case characters required.            |
+| `complexity.minNumbers`      | Integer                                                                                                            | Minimum number of numerical characters required.             |
+| `complexity.minSymbols`      | Integer                                                                                                            | Minimum number of symbols required.                          |
+| `complexity.minUpperCase`    | Integer                                                                                                            | Minimum number of upper case characters required.            |
+| `created`                    | String (ISO-8601)                                                                                                  | Timestamp when the Factor Profile feature was created.       |
+| `exclude.attributeCriteria`  | Array (Attribute Criterion Object) | Attribute criteria settings (Required but empty allowed).    |
```

Comment:
Link for Attribute Criterion Object missing?

Also since this is the response body, no need to specify "(Required but empty allowed)" here